src/platform_impl/android/keycodes.rs

```diff
@@ -156,324 +159,393 @@ pub fn to_physical_keycode(keycode: Keycode) -> KeyCode {
+                F3 => Key::F3,
+                F4 => Key::F4,
+                F5 => Key::F5,
+                F6 => Key::F6,
+                F7 => Key::F7,
+                F8 => Key::F8,
+                F9 => Key::F9,
+                F10 => Key::F10,
+                F11 => Key::F11,
+                F12 => Key::F12,
+                NumLock => Key::NumLock,
+                Numpad0 => Key::Character("0".into()),
+                Numpad1 => Key::Character("1".into()),
+                Numpad2 => Key::Character("2".into()),
+                Numpad3 => Key::Character("3".into()),
+                Numpad4 => Key::Character("4".into()),
+                Numpad5 => Key::Character("5".into()),
+                Numpad6 => Key::Character("6".into()),
+                Numpad7 => Key::Character("7".into()),
+                Numpad8 => Key::Character("8".into()),
+                Numpad9 => Key::Character("9".into()),
+                NumpadDivide => Key::Character("/".into()),
+                NumpadMultiply => Key::Character("*".into()),
+                NumpadSubtract => Key::Character("-".into()),
+                NumpadAdd => Key::Character("+".into()),
+                NumpadDot => Key::Character(".".into()),
+                NumpadComma => Key::Character(",".into()),
+                NumpadEnter => Key::Enter,
+                NumpadEquals => Key::Character("=".into()),
+                NumpadLeftParen => Key::Character("(".into()),
+                NumpadRightParen => Key::Character(")".into()),
+
+                VolumeMute => Key::AudioVolumeMute,
+                Info => Key::Info,
+                ChannelUp => Key::ChannelUp,
+                ChannelDown => Key::ChannelDown,
+                ZoomIn => Key::ZoomIn,
+                ZoomOut => Key::ZoomOut,
+                Tv => Key::TV,
+                Window => Key::Unidentified(native),
+                Guide => Key::Guide,
+                Dvr => Key::DVR,
+                Bookmark => Key::BrowserFavorites,
+                Captions => Key::ClosedCaptionToggle,
+                Settings => Key::Settings,
+                TvPower => Key::TVPower,
+                TvInput => Key::TVInput,
+                StbPower => Key::STBPower,
+                StbInput => Key::STBInput,
+                AvrPower => Key::AVRPower,
+                AvrInput => Key::AVRInput,
+                ProgRed => Key::ColorF0Red,
+                ProgGreen => Key::ColorF1Green,
+                ProgYellow => Key::ColorF2Yellow,
+                ProgBlue => Key::ColorF3Blue,
+                AppSwitch => Key::AppSwitch,
+                Button1 => Key::Unidentified(native),
+                Button2 => Key::Unidentified(native),
+                Button3 => Key::Unidentified(native),
+                Button4 => Key::Unidentified(native),
+                Button5 => Key::Unidentified(native),
+                Button6 => Key::Unidentified(native),
+                Button7 => Key::Unidentified(native),
```

Comment:
Could you move all `Unidentified` stuff to the bottom of the `match`?

Reply:
guess so

Reply:
really though - this is just an existing match that's been indented, this is busy work that's not really important surely?

Reply:
dunno, your editor should allow that to do in a couple keystrokes. I can format it myself under 5-10s.

Reply:
it's also notable that they are currently in numerical order and some of them are logically related to other supported keys that are adjacent. I think we should leave these ordered as they are.

Reply:
Maybe, it's just hard to grasp what really is mapped and what's not. If the thing is not mapped it doesn't really matter, so keeping it separately in the end is more pleasant to read.